src/main/java/kr/hhplus/be/server/Exception/EcommerceException.java

```diff
@@ -0,0 +1,21 @@
+package kr.hhplus.be.server.Exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class EcommerceException extends RuntimeException {
+
+    private String code;
+    private String message;
+
+    public EcommerceException(ErrorCode errorCode) {
+        this.code = errorCode.getCode();
+        this.message = errorCode.getMessage();
```

Comment:
부모 클래스에도 message값을 설정해주면 스택 트레이스 로깅에 유용합니다.
```suggestion
        super(errorCode.getMessage());
        this.code = errorCode.getCode();
        this.message = errorCode.getMessage();
```